src/App/pages/CategoriesPage/components/CategoryCard/CategoryCard.tsx

```diff
@@ -1,33 +1,34 @@
 import React, { useState } from 'react';
 import classNames from 'classnames';
-import { Text, Placeholder } from 'components';
+import { Text } from 'components';
+import { CategoryApi } from 'api/types';
+import getFixedFallbackImage from 'utils/getFallbackImage';
+
 import styles from './CategoryCard.module.scss';
 
 export type CategoryCardProps = {
   className?: string;
-  image: string;
-  name: React.ReactNode;
   onClick?: React.MouseEventHandler;
+  category: CategoryApi;
 };
 
 const CategoryCard: React.FC<CategoryCardProps> = (props) => {
-  const { className, image, name, onClick } = props;
-
+  const { className, onClick, category } = props;
   const [imgError, setImgError] = useState(false);
 
   return (
     <div className={classNames(styles.card, className)} onClick={onClick}>
-      {image && !imgError ? (
-        <div className={styles.imgContainer}>
-          <img className={styles.img} src={image} alt="Category" onError={() => setImgError(true)} />
-        </div>
-      ) : (
-        <Placeholder height={260} />
-      )}
+      <div className={styles.imgContainer}>
+        {category.image && !imgError ? (
+          <img className={styles.img} src={category.image} alt="Category" onError={() => setImgError(true)} />
+        ) : (
+          <img className={styles.img} src={getFixedFallbackImage(category.name, undefined)} />
+        )}
```

Comment:
вот так по идее тоже должно работать
```suggestion
        <img
          className={styles.img}
          src={category.image || fallback}
          alt="Category"
          onError={(e) => {
            e.currentTarget.src = fallback;
          }}
        />
```